.nuke/build.schema.json

```diff
@@ -33,6 +33,7 @@
             "AppVeyor",
             "AzurePipelines",
             "Bamboo",
+            "Bitbucket",
```

Comment:
I ran the tests through nuke and it did this 🤷

Reply:
Does this for me as well.